hack/testing/README.md

```diff
@@ -38,7 +38,47 @@ to have been created (based on the logs of the Elasticsearch pods).
 This Go script will query the appropriate ES pod (if an OPS cluster was used)
 and check the local log files for the each result message.
 
-## check-curator.sh
+## Environment values for testing
+
+In addition to providing the argument `true` to `e2e-test.sh` to indicate that
+an OPS cluster is used, the following environment values can be used to
+change behavior of the scripts.
+
+The format for the following would be
+`ENV_VAR1=value [ENV_VAR2=value2] ./e2e-test.sh`
+
+```
+VERBOSE                 | If set, the scripts will print out each command as executed | default: "" (unset)
+KIBANA_CLUSTER_SIZE     | The num of Kibana pod replicas | default: 1
+KIBANA_OPS_CLUSTER_SIZE | The num of Kibana Ops pod replicas if OPS cluster is used | default: 1
+ES_CLUSTER_SIZE         | The num of Elasticsearch pod replicas | default: 1
+ES_OPS_CLUSTER_SIZE     | The num of Elasticsearch Ops pod replicas if OPS cluster is used | default: 1
+TIMES                   | The maximum number of attempts to try for checking if ES is ready for queries | default: 10
+QUERY_SIZE              | The maximum number of query results returned for an index | default: 500
+```
+
+# logging.sh
+The script logging.sh is the main CI test driver.  There are a number of
+environment variables that can control its behavior.  In addition, giving the
+`NOSETUP` argument to the script will cause it to not set up OpenShift and the
+logging components.  Instead, it will assume these have already been set up,
+and will just configure the environment to run the tests, then run the tests.
+```
+USE_JOURNAL             | If set to "true" or "false", explicitly enable or disable using the journald as the logging source | default: unset - fluentd will use whatever is configured for the system
```

Comment:
We should separate arguments like this that apply only to startup from those like the Operations Cluster stuff which can be added post-facto